packages/@webex/plugin-cc/src/cc.ts

```diff
@@ -136,4 +138,35 @@ export default class ContactCenter extends WebexPlugin implements IContactCenter
       return Promise.reject(error);
     }
   }
+
+  /**
+   * @param options - GetBuddyAgentsOptions
+   * @returns Promise<BuddyAgentsResponse>
+   * @throws Error
+   * @example
+   * const buddyAgents = await webex.cc.getBuddyAgents({
+   *  agentProfileId: 'selfAgentProfileId',
+   *  mediaType: BUDDY_AGENT_MEDIA_TYPE.TELEPHONY,
+   *  state: BuddyAgentState.AVAILABLE,
+   *  });
+   */
+  public async getBuddyAgents({
```

Comment:
Why have we created 3 functions for this?

Reply:
Discussed 1:1